build.gradle

```diff
@@ -202,8 +202,3 @@ buildscript {
         classpath 'org.eclipse.jgit:org.eclipse.jgit:2.0.0.201206130900-r'
     }
 }
-
-task wrapper(type: Wrapper) {
```

Comment:
Gotcha, thanks! Honestly, I perform the gradle upgrade so rarely that I'm never going to be able to remember the exact parameter names, etc. Let's just keep the task there, for future convenience :)